.github/workflows/doc-build.yml

```diff
@@ -95,28 +90,14 @@ jobs:
       script: |
         set -euo pipefail
 
-        REF_TYPE=${{ github.ref_type }}
```

Comment:
Please add a comment to the top of this block describing the intended behavior, like what you included in the PR description. It'll be helpful for future readers to understand what this logic is supposed to do.